### DISCUSSION TITLE
setting up the environment of finn in ubuntu,encountered an issue,any ideas why?

### DISCUSSION THREAD

**Discussion by harrisonasd** [Xilinx/finn#985] [Q&A] [answered]
during the final step (bash ./run-docker.sh notebook) of setting up the environment,an error occurred,and the process was terminated.does anyone know why?

ImportError: cannot import name 'AttrsInstance' from 'attr' (/tmp/home_dir/.local/lib/python3.8/site-packages/attr/__init__.py)
ERROR: After October 2020 you may experience errors when installing or updating packages. This is because pip will change the way that it resolves dependency conflicts.

We recommend you use --use-feature=2020-resolver to test your packages with the new resolver before it becomes the default.

referencing 0.33.0 requires attrs>=22.2.0, but you'll have attrs 19.3.0 which is incompatible.
jsonschema 4.21.1 requires attrs>=22.2.0, but you'll have attrs 19.3.0 which is incompatible.

**Comment by aidanrhind** [comment]
Facing the same issue 

Answer was found in this discussion post: Switch to dev branch
https://github.com/Xilinx/finn/discussions/854

**Comment by harrisonasd** [comment]
I have set this up, but there are still errors above.
FINN_XILINX_PATH = /tools/Xilinx
FINN_XILINX_VERSION = 2022.2

I'm not sure if it's because my Vivado version is 2022.1.

**Comment by harrisonasd** [comment]
![image](https://github.com/Xilinx/finn/assets/105301827/082015af-8dc6-4546-aac4-ecf478b43c2a)
Have you encountered this situation before

**Reply by fpjentzsch** [reply, marked as answer]
Did you install only Vivado and Vitis HLS, but not the full Vitis suite? Vitis is only required for Alveo builds, so you should have no problems if you are targeting regular PYNQ boards.

**Reply by harrisonasd** [reply]
thank you so much
